tslint.json

```diff
@@ -68,7 +68,6 @@
     "prefer-type-cast": false,
     "promise-must-complete": true,
     "underscore-consistent-invocation": true,
-    "use-named-parameter": true,
-    "valid-typeof": true
```

Comment:
This is now handled by the compiler: https://github.com/Microsoft/TypeScript/pull/13791